src/main/java/dev/rndmorris/salisarcana/config/group/ConfigBugfixes.java

```diff
@@ -245,6 +245,24 @@ public class ConfigBugfixes extends ConfigGroup {
         "clampWandOverlayVis",
         "Prevents the \"Vis in Wand\" GUI when the wand is held from showing impossible bars when the wand holds impossible amounts of vis.");
 
+    public final ToggleSetting detectZeroAspectBlocks = new ToggleSetting(
+        this,
+        "detectZeroAspectBlocks",
+        "Fixes an oversight with the pickaxe of the core that prevents it from displaying detected blocks that have no aspects.")
+            .setCategory("tools");
+
+    public final ToggleSetting detectLitRedstoneOre = new ToggleSetting(
+        this,
+        "detectLitRedstoneOre",
+        "Works around limitation with Forge that prevents the pickaxe of the core fom detecting lit redstone ore by treating it\n"
+            + "as identical to regular redstone ore.").setCategory("tools");
+
+    public final ToggleSetting detectLapisOre = new ToggleSetting(
+        this,
+        "detectLapisOre",
+        "Allows the pickaxe of the core to correctly detect vanilla lapis ore.\n" + "\n"
+            + "Fixes a bug with the pickaxe of the core that caused it to rely on the damage value of a block's dropped item to\n"
+            + "identify a block instead of its in-world metadata when scanning for ores.").setCategory("tools");
```

Comment:
Consider using triple-ticked multiline strings instead?

```suggestion
        """
            Allows the pickaxe of the core to correctly detect vanilla lapis ore.
            
            Fixes a bug with the pickaxe of the core that caused it to rely on the damage value of a block's dropped item to
            identify a block instead of its in-world metadata when scanning for ores.""").setCategory("tools");
```